### ISSUE TITLE
SyncFusion DLL 

### ISSUE THREAD

**Issue by AlexF12004Roma** [hansworst24/Ticz#26]
Hi,

I'm starting using Domoticz and I found this wonderful UWP app.
I see that you stopped  development so I'd like to download and build the project for fix/add things i will need during my future use.

I see you used SyncFusion Lib but in git project they are not included. Now SyncFusion seems not free.
Did you use an old but free version? if yes, can you share?

Thank you.

**Comment by hansworst24** [comment]
Hi Alex,

great to hear you like the project :) I went to Android due to lack of confidence in the mobile platform, but it has to be said I think Ticz is way better/nicer than the Android alternatives.... :) So currently I'm looking into developing Ticz for the Android platform using Xamarin Forms, which also enables me to cross-platform it as UWP.... but that project will take a long, long time... :)

The Syncfusion stuff is on a community license which you can get from https://www.syncfusion.com/products/communitylicense. You have to register, but then you're able to use all products for free, as long as you use it in open-source stuff ! Alternatively you just rip out the Syncfusion stuff (which by memory is only used within the graphs section).

Have fun rebuilding and modifying it as you like ! Don't blame me for the bad coding :) It was just a hobby project :)

https://www.syncfusion.com/products/communitylicense

**Comment by AlexF12004Roma** [comment]
OK, thank you!

You can use VisualStudio 2015 for biulding Xamarin cross platform app.
You can resuse most of the code from Ticz, i think it will take me much less time to develop.
You should use C#.NET instead of VB.NET but it's better.
Ale.

**Comment by hansworst24** [comment]
Yup, that's the plan :) Translating the code behind from VB to C# won't be the main issue, although I never wrote C#. Getting the GUI look the same with Xamarin.Forms as the UWP version will be challenging. There isn't yet a full stable previewer available in VS2015 for Xamarin.Forms, so the look and feel will take a lot of time :)
